modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/helper/DDMFormEvaluatorHelperTest.java

```diff
@@ -266,6 +269,9 @@ public void testDDMFormPageChange() throws Exception {
 		ddmFormLayout.setNextPage(1);
 		ddmFormLayout.setPreviousPage(0);
 
+		ddmFormLayout.addDDMFormLayoutPage(new DDMFormLayoutPage());
+		ddmFormLayout.addDDMFormLayoutPage(new DDMFormLayoutPage());
+
```

Comment:
@linolaoj Why do we need these two lines?

And I believe that `addDDMFormLayoutPage` can come before `setNextPage`.

Reply:
@carolmariaabb , because this fix checks the pages, if we go from page 0 to page 1, the form needs to have (at least) 2 pages.
Yes the `addDDMFormLayoutPage` can come before `setNextPage`. I'll make this change.